api/storage/v1alpha1/vulnerabilityreport_types.go

```diff
@@ -18,7 +18,18 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type Class string
+
+// Enumeration of supported package classes
+const (
+	// Operating system packages (e.g., deb, rpm)
+	OSPackagesClass = "os-pkgs"
```

Comment:
Please use `Class` as a prefix:
```go
ClassOSPackages
...
ClassBinary
```

NOTE: We have inconsistent constant naming throughout the codebase, but we agreed to use the type as a prefix for constants since it's more ergonomic. I'll update the rest of the code to follow this convention.